homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -181,6 +191,23 @@ def put(self, request, username, entity_number):
             if entity_features & SUPPORT_BRIGHTNESS:
                 if brightness is not None:
                     data[ATTR_BRIGHTNESS] = brightness
+            if entity_features & SUPPORT_XY_COLOR:
+                if color is not None:
+                    data[ATTR_XY_COLOR] = color
+            elif entity_features & SUPPORT_RGB_COLOR:
+                if color is not None:
+                    if brightness is not None:
```

Comment:
This part requires testing. https://coveralls.io/builds/10389271/source?filename=homeassistant%2Fcomponents%2Femulated_hue%2Fhue_api.py#L199